Chapters/1-5-Chapter5.md

```diff
@@ -0,0 +1,155 @@
+
+"No one is permitted in the Senate while they are deliberating."
+
+"Yes, I understand. I am delivering it to her assistant."
+
+"She hasn't mentioned this to us."
+
+"Yes. I was summoned by the speaker, as the matter is unexpected and urgent."
+
+She deliberated with her partner, and glanced at me once, sending a shock of worry through my bones. I was getting cold again.
+
+"You are permitted to go, but your attendant cannot come with you."
+
+_NO_
+
+He replied, "My... attendant helps me walk on account of my injured knee. I can't go anywhere without help these days."
+
+The male guard begrudgingly walked over to me and said, "Open your bag."
+
+I complied and he rummaged for a bit before walking back and giving a hand sign to the other guard. She said, "Go ahead."
+
+The senate grounds were surrounded by the building that the rest of the city called the Senate. It was a massive circle of offices and gates and walkway, its diameter as large as the farmland I worked in Heiko. But the true focus of this place was the deliberation hall: a giant marble sphere standing proudly in the center, illuminated by the light through Ara's gate. It sat upon a short marble pedastal that seemed too small to be holding so much weight. Between the outer circle and the sphere was a labyrinth of small pools and lines of vegetation that could certainly not live in such a hot and dry place without the constant shade that the perimeter circle granted. At every vertex of the planting lines, a small black iron circle with an engraved crest marked a city that was member to the Shaki Senate. I longed to venture through the plants and learn about them, but I dared not draw attention to myself.
+
+Fajro set me back on track, "Perhaps it's best that we head towards the Heiko office so as not to seem suspicious. We'll want to enter through that side anyway."
+
+I remained silent and compliant, too nervous to say anything. We followed the inside of the perimeter circle to the right into the Heiko section, and turned the corner to face the deliberation hall. It was more shaded from this side, but still as radiant as the rest of the city. There were two guards at the doors again with the same woven armor and massive rectangular shields built to repel several people at once.
+
+"How can we get through them?" I asked, feeling doubtful.
+
+"Leave that to me. This is why Kipnu sent me after all. I'll distract them while you go around and slip through the door."
+
+A pang of anger welled up in me but I kept my eyes faced forward, "What sort of plan is that?"
+
+"Oh I think you'll find I can be very distracting."
+
+"Anyone can be distracting. You're a fidelant. Why not use that to get me in?"
+
+"The power of law lies in its enforcement. And trust me when I say these guards know enforcement. Not even I can wrest them from their duty to the law."
```

Comment:
```suggestion
"The power of law lies in its enforcement. And trust me when I say these guards know enforcement. Not even I can wrestle them from their duty to the law."
```

Reply:
wrest is a word, and I think I prefer it here.